### ISSUE TITLE
LOT_SIZE error can't resolve..:(

### ISSUE THREAD

**Issue by MuhammadNouman** [jaggedsoft/php-binance-api#38]
I am new to binance API, placing a buy order but can't resolve the error of LOT_SIZE
My values are within limits of minQty and MIN_notional
Below are my values for example, 
//Asset = CDTBTC

`0.00000001 Min Price
100000.00000000 Max Price
1.00000000 Min Quantity
90000000.00000000 MaxQty
1.00000000 Step Size
0.00200000 MiNotional

Quantity of CDTBTC to be traded: 2.98786828`

$api->buy($symbol, $quantity, $price)

**Comment by jaggedsoft** [comment]
minNotional is .002
So this means you must order .002btc worth of CDT at minimum

last price of CDT is 0.00001150
divide minNotional by the price: .002 / 0.00001150
`amount = minNotional / lastPrice;`
equals 173.91304347826087 units, the minimum you must buy to get .002 btc
roundStep is 1, so you must call `Math.ceil(amount)` to round it to 174

**Comment by MuhammadNouman** [comment]
ok, thanks for quick reply,another thing, from where I can get the latest updated price of an asset using websocket? i.e which websocket endpoint can b used to have a continous look at latest price of an asset

**Comment by jaggedsoft** [comment]
I just issued a new release with my favorite function

#### Get ticker updates for all symbols via WebSocket
```php
$api->ticker(false, function($api, $symbol, $ticker) {
	print_r($ticker);
});
```

#### Get ticker updates for a specific symbol via WebSocket
```php
$api->ticker("CDTBTC", function($api, $symbol, $ticker) {
	print_r($ticker);
});
```

**Comment by jaggedsoft** [comment]
Available in `v0.0.12`
